commits.go

```diff
@@ -121,6 +131,31 @@ type CommitAction struct {
 	Encoding     string     `url:"encoding,omitempty" json:"encoding,omitempty"`
 }
 
+// GetCommitRefs Get all references (from branches or tags) a commit is pushed to
+// _type can be one of "all, branch, tag"
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/commits.html#get-references-a-commit-is-pushed-to
+func (s *CommitsService) GetCommitRefs(pid interface{}, sha string, _type string, options ...OptionFunc) ([]CommitRef, *Response, error) {
```

Comment:
The approach in this package is to use method parameters for all variables that go into the path (URL) and option struct containing all the parameters that go into body. So in this case you need to make options struct for the `_type` variable.

And since this endpoint also supports pagination, you should then also embed the `ListOptions` struct.

Last but not least, please don't use `_type` as a variable name :wink: And since only a few specific values are allowed, we prefer to make a specific type for that so people cannot make a mistake by supplying a non-valid value.